pull/github.go

```diff
@@ -185,6 +185,37 @@ func (ghc *GitHubContext) Title() string {
 	return ghc.pr.Title
 }
 
+type v4PullRequestWithEditedAt struct {
+	Author       v4Actor
+	CreatedAt    time.Time
+	LastEditedAt time.Time
+	Body         string
+}
+
+func (ghc *GitHubContext) Body() (*Body, error) {
+	var q struct {
+		Repository struct {
+			PullRequest v4PullRequestWithEditedAt `graphql:"pullRequest(number: $number)"`
+		} `graphql:"repository(owner: $owner, name: $name)"`
+	}
+	qvars := map[string]interface{}{
+		"owner":  githubv4.String(ghc.owner),
+		"name":   githubv4.String(ghc.repo),
+		"number": githubv4.Int(ghc.number),
+	}
+	if err := ghc.v4client.Query(ghc.ctx, &q, qvars); err != nil {
+		return &Body{}, errors.Wrap(err, "failed to load pull request details")
```

Comment:
nit: since this function returns a pointer, let's return `nil` on error instead of an empty struct

```suggestion
		return nil, errors.Wrap(err, "failed to load pull request details")
```